testcase/cross-region/tso.go

```diff
@@ -70,17 +70,17 @@ func (c *crossRegionClient) testTSO(ctx context.Context) error {
 		return err
 	}
 	log.Info("new leader ready")
-	for i := 1; i <= dcLocationNumber; i++ {
-		if err := c.transferPDAllocator(fmt.Sprintf("dc-%d", i)); err != nil {
+	for i := 0; i < dcLocationNumber; i++ {
+		if err := c.transferPDAllocator(DCLocations[i]); err != nil {
 			return err
 		}
 	}
 	log.Info("new allocators ready")
 	// after transfer allocator and leader, we can tolerate tso failed once for each dc.
 	c.requestTSOAfterTransfer(ctx, "global")
-	c.requestTSOAfterTransfer(ctx, "dc-1")
-	c.requestTSOAfterTransfer(ctx, "dc-2")
-	c.requestTSOAfterTransfer(ctx, "dc-3")
+	c.requestTSOAfterTransfer(ctx, DCLocations[0])
+	c.requestTSOAfterTransfer(ctx, DCLocations[1])
+	c.requestTSOAfterTransfer(ctx, DCLocations[2])
```

Comment:
I think using a loop here to iter is better.

Reply:
Ack.